apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -24,13 +27,21 @@ const Auth = (): JSX.Element => {
 	const authenticatedUser = useAppSelector(
 		(state) => state.auth.authenticatedUser,
 	);
+	const userPermissions = useMemo(() => {
+		return authenticatedUser
+			? authenticatedUser.groups.flatMap((group) => group.permissions)
+			: [];
+	}, [authenticatedUser]);
+
 	const navigate = useNavigate();
 
 	useEffect(() => {
 		if (authenticatedUser) {
-			navigate(AppRoute.ROOT);
+			navigate(
+				getFirstAvailableRoute(userPermissions, SIDEBAR_ITEMS, AppRoute.ROOT),
+			);
```

Comment:
let's navigate to the projects here, and handle redirect to the first available item inside ProtectedRoute
```suggestion
			navigate(
				navigate(AppRoute.PROJECTS);
			);
```